### PR TITLE
Add tranable for CVS and Multi

### PR DESCRIPTION
**Pull request by hojin-choi**
Transaction APIs by credit card and convenience store are necessary in my project and this module is really helpful for me.
But it was not implemented about transaction for CVS.
- EntryTranCvs.idPass
- ExecTranCvs.idPass
- SearchTradeMulti.idPass

So, I added 3 APIs.

Additionally, There were 2 problems when I run test command.
1. `ava-ts` requires a peer of `ava@0.x` but you updated ava@1.x.. I think `ava-ts` could be replaced with `ava` and configuration for typescript.
2. When I run test case with updated `ava@1.x`, there was an error. I'm not sure but I thought this error related with typescript version.
```
node_modules/ava/index.d.ts(406,106): error TS1110: Type expected.
node_modules/ava/index.d.ts(415,112): error TS1110: Type expected.
```

Please check it and I hope you to publish these APIs if you don't have a plan about these features.

### CODE REVIEW

File: src/client/tranableCvs.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import Client from '../client'
+import {IConfig} from '../config.interface'
+import {
+  IEntryTranCvsArgs,
+  IEntryTranCvsResult,
+  IExecTranCvsArgs,
+  IExecTranCvsResult
+} from './tranableCvs.interface'
+
+export default class TranableCvs extends Client {
```
[the remark this answers is not in the document]
Comment (hojin-choi) [reply src/client/tranableCvs.ts]: I agree with your opinion and I have a question.
I'm gonna change `TranableCvs` class name to `CvsTranable` as you referred, what do you think about `TranableMulti` class name?
`TranableMulti` class is also mixed in to the main class, so should this class name be `MultiTranable`?

[the remark this answers is not in the document]
Comment (linyows) [reply src/client/tranableCvs.ts]: correct!

**Comment by hojin-choi** [comment]
Additionally, I attach the encoder for post body for encoding the character set `shift_jis`.

Because when I request the transaction for the convenience store with argument `CustomerName` and `CustomerKana` with Japanese character, GMO PG server response with error code `M01010013` or `M01011013`.

I read the GMO PG system manual document No.035 and No.021, it refers to use `shift_jis` code system for the use of Japanese.
So I think the body data on the post request is necessary to encoding to `shift_jis`.

Please check it. Thank you.

**Comment by linyows** [comment]
https://faq.gmo-pg.com/service/detail.aspx?id=1384

Oops... I understood that a shift_jis encoder is necessary. Thanks. 👍

**Comment by hojin-choi** [comment]
Thank you for approve this PR.
Please publish this module on NPM when you have a time :)

**Comment by linyows** [comment]
I released this PR  on npm as v1.0.0. Thank you.

**Pull request merged**